presto-docs/src/main/sphinx/functions/comparison.rst

```diff
@@ -153,3 +153,44 @@ Expression              Meaning
 ====================    ===========
 
 ``ANY`` and ``SOME`` have the same meaning and can be used interchangeably.
+
+LIKE
+-----
+The LIKE operator is used in a WHERE clause to search for a specified character pattern in a string. Pattern matching can be done on regular characters as well as wildcards. Wildcard characters can be escaped. Matching is case sensitive.
```

Comment:
It's a general boolean expression. So it doesn't have to be in a WHERE clause. Also make it more concise and explicit by saying "Patterns can contain regular characters as well as wild card..."